doc/conf.py

```diff
@@ -0,0 +1,78 @@
+"""
+Configuration file for the Sphinx documentation builder.
+
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+
+# pylint: disable = redefined-builtin, invalid-name
+
+import os
+import sys
+
+from rflx import __version__
+
+# -- Path setup --------------------------------------------------------------
+
+sys.path.insert(0, os.path.abspath(".."))
+
+# -- Project information -----------------------------------------------------
+
+project = "RecordFlux"
+copyright = "2022, AdaCore GmbH"
+author = "AdaCore GmbH"
```

Comment:
Please change to just "AdaCore" (omit GmbH).

Reply:
Changed.